### ISSUE TITLE
If startup fails because of Tor show restart/refresh Tor button

### ISSUE THREAD

**Issue by suddenwhipvapor** [bisq-network/bisq2#3050]
It has been several times already that users report seeing all offers at 0 reputation score, even by known reputable sellers, and all users at 0 score under Reputation > Ranking.
I usually suggest they restart Bisq, maybe also deleting the tor folder in the data directory, but that requires the users to identify the issue, report it in the chat, wait for a reply, and then fumbling with the application.

I think adding a check to verify if there are reputable sellers in the ranking window, or not, after enough peers were connected, and prompting user to take action, is a nice feature to add.
Ideally, the user would see a short popup explaining what's up, and a "Press to reset tor" button that empties the tor folder and closes bisq2, so the user can restart it manually.

**Comment by HenrikJannsen** [comment]
Can you request log when that happens? Did they see the network data state in the left lower bottom being green? There is a text indicating state of loaded data.

**Comment by suddenwhipvapor** [comment]
that's right, I erroneously supposed all was good because a coloured text is shown down there if it isn't... but you never suppose with other people 😅 
Will require logs next time

**Comment by HenrikJannsen** [comment]
We should handle a failed tor startup which causes the timeout exception at startup by showing the user a popup with info that tor startup failed and instruction to shut down and restart the app.

**Comment by HenrikJannsen** [comment]
New loading screen will show delete tor and restart button after 2 minutes.
